client/models/saved_searches.go

```diff
@@ -12,142 +12,144 @@ type SavedSearchesEntry struct {
+	ActionEmailMaxTime                    string  `json:"action.email.maxtime,omitempty" url:"action.email.maxtime,omitempty"`
+	ActionEmailMessageAlert               string  `json:"action.email.message.alert,omitempty" url:"action.email.message.alert,omitempty"`
+	ActionEmailMessageReport              string  `json:"action.email.message.report,omitempty" url:"action.email.message.report,omitempty"`
+	ActionEmailPDFView                    string  `json:"action.email.pdfview,omitempty" url:"action.email.pdfview,omitempty"`
+	ActionEmailPreprocessResults          string  `json:"action.email.preprocess_results,omitempty" url:"action.email.preprocess_results,omitempty"`
+	ActionEmailReportCIDFontList          string  `json:"action.email.reportCIDFontList,omitempty" url:"action.email.reportCIDFontList,omitempty"`
+	ActionEmailReportIncludeSplunkLogo    bool    `json:"action.email.reportIncludeSplunkLogo" url:"action.email.reportIncludeSplunkLogo"`
+	ActionEmailReportPaperOrientation     string  `json:"action.email.reportPaperOrientation,omitempty" url:"action.email.reportPaperOrientation,omitempty"`
+	ActionEmailReportPaperSize            string  `json:"action.email.reportPaperSize,omitempty" url:"action.email.reportPaperSize,omitempty"`
+	ActionEmailReportServerEnabled        bool    `json:"action.email.reportServerEnabled" url:"action.email.reportServerEnabled"`
+	ActionEmailReportServerURL            string  `json:"action.email.reportServerURL,omitempty" url:"action.email.reportServerURL,omitempty"`
+	ActionEmailSendCSV                    int     `json:"action.email.sendcsv,string,omitempty" url:"action.email.sendcsv,omitempty"`
+	ActionEmailSendPDF                    bool    `json:"action.email.sendpdf" url:"action.email.sendpdf"`
+	ActionEmailSendResults                bool    `json:"action.email.sendresults" url:"action.email.sendresults"`
+	ActionEmailSubject                    string  `json:"action.email.subject,omitempty" url:"action.email.subject,omitempty"`
+	ActionEmailTo                         string  `json:"action.email.to,omitempty" url:"action.email.to,omitempty"`
+	ActionEmailTrackAlert                 bool    `json:"action.email.track_alert" url:"action.email.track_alert"`
+	ActionEmailTTL                        string  `json:"action.email.ttl,omitempty" url:"action.email.ttl,omitempty"`
+	ActionEmailUseSSL                     bool    `json:"action.email.use_ssl" url:"action.email.use_ssl"`
+	ActionEmailUseTLS                     bool    `json:"action.email.use_tls" url:"action.email.use_tls"`
+	ActionEmailWidthSortColumns           bool    `json:"action.email.width_sort_columns" url:"action.email.width_sort_columns"`
+	ActionPagerdutyIntegrationURL         string  `json:"action.pagerduty.param.integration_url,omitempty" url:"action.pagerduty.param.integration_url"`
+	ActionPagerdutyIntegrationURLOverride string  `json:"action.pagerduty.param.integration_url_override,omitempty" url:"action.pagerduty.param.integration_url_override"`
```

Comment:
These are the new fields. The long ActionPagerdutyIntegrationURLOverride moved all other stuff to the right.